packages/wrangler/src/miniflare-cli/assets.ts

```diff
@@ -4,10 +4,7 @@ import { createMetadataObject } from "@cloudflare/pages-shared/metadata-generato
 import { parseHeaders } from "@cloudflare/pages-shared/metadata-generator/parseHeaders";
 import { parseRedirects } from "@cloudflare/pages-shared/metadata-generator/parseRedirects";
 import { fetch as miniflareFetch } from "@miniflare/core";
-import {
-	Response as MiniflareResponse,
-	Request as MiniflareRequest,
-} from "@miniflare/core";
+import { Response as MiniflareResponse } from "@miniflare/core";
```

Comment:
Need to replace this (and probably a few others) with the Fetch API things newly exported from `@miniflare/tre`) (https://github.com/cloudflare/miniflare/pull/440).